store/indexStore.ts

```diff
@@ -0,0 +1,14 @@
+import create from 'zustand'
+import {  ZERO } from '../constants/numbers'
+
+interface IndexState {
+    ethDvolIndex: Number
```

Comment:
Can we move this to Appstrore or crabV2store instead of creating new one?

Reply:
Okay...moved this to crabv2